Prace_domowe/Praca_domowa2/Chylak_Maciej/Chylak_Maciej_PD2.ipynb

```diff
@@ -0,0 +1,2914 @@
+{
```

Comment:
<p>Aby określić czy błąd jest duży, zazwyczaj RMSE odnosimy to średniej wartości w danych</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/110/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>